### ISSUE TITLE
Hardware List

### ISSUE THREAD

**Issue by duncanwilkinson** [galczo5/open-midi-controller#6]
Hi @galczo5 this is incredible! Thank you so much for posting. Could you share some direct links next to the components required please? This will be my first Arduino project so want to make sure I order all the right parts. Thank you again!

**Comment by galczo5** [comment]
Yeah, sure. I can try to find them and post the list here. Aliexpress is ok with you?

**Comment by duncanwilkinson** [comment]
Yeah that would be great, thank you again.

**Comment by galczo5** [comment]
Arduino: https://www.aliexpress.com/item/1005001706390728.html

Shield (optional if you don't want to solder it directly to the board): https://www.aliexpress.com/item/1005002867053794.html

Momentary switches: https://www.aliexpress.com/item/4000606891227.html

LCD: https://www.aliexpress.com/item/32413056677.html

DIN socket (for midi DIN 5): https://www.aliexpress.com/item/1005003195908920.html

**Comment by duncanwilkinson** [comment]
Thank you so much @galczo5 that's really, really helpful. Now just to find a cool enclosure for it, maybe a metal box with a Flux Capacitor sign for the faceplate: https://www.aliexpress.com/item/1005003642997752.html :)

**Comment by Versus93** [comment]
Hello @galczo5, could you please provide a measure for the enclosure? Would Hammond size 1590DD fit?

**Comment by galczo5** [comment]
I think that 1590DD might be slightly too small. It depends on your skill in cable management. 

I used a generic metal case from an electric store. It's a little bit bigger than 1590dd, like 2-5cm, but it's higher and it allowed me to fit it nicely. 

Aluminum cases like 1590 might be problematic to drill a screen hole, because of thick aluminum sides
